block_designs.tex

```diff
@@ -1,6 +1,144 @@
 \chapter{Blokové plány}
 
+Blokové plány sú kombinatorické štruktúry, pre ktoré existuje veľa využití v štatistike, dizajne experimentov, teórii kódovania, teórii grafov a ďalších oblastiach. Napríklad aj pri dizajne spoločenských hier :) Populárna hra Dobble je vlastne blokový plán, kde jednotlivé karty sú bloky a symboly na kartách sú objekty.
+
+\section{Symetrické blokové plány}
+
+\begin{definition}
+
+Nech $X = \set{x_1, \ldots, x_v}$ je množina objektov a $\mathcal{B} = \set{X_1, \ldots, X_v} \subset \powerset(X)$ je množina podmnožín objektov (tieto podmnožiny voláme \emph{bloky}), pričom sú splnené nasledujúce podmienky:
+\begin{enumerate}
+	\item $|X_i| = k$ pre $i = 1, 2, \ldots, v$
```

Comment:
There should be semicolons (;) at the end of the items and dot at the end of the last item.